vpdq/python/README.md

```diff
@@ -0,0 +1,45 @@
+# vpdq
+This is a Python binding library to the vPDQ (video PDQ) hashing algorithm by Facebook. The project is at [github](https://github.com/facebook/ThreatExchange/tree/main/vpdq)
+
+## Installation
+### Install library
+```
+pip install --upgrade pip
+pip install vpdq
+```
+### Install FFmpeg
+
+Before using VPDQ to create hashes, FFmpeg must be installed. (Easier to use if Accessible via the `$PATH` environment variable)
+
+There are a variety of ways to install FFmpeg, such as the [official download links](https://ffmpeg.org/download.html), or using your package manager of choice (e.g. `sudo apt install ffmpeg` on Debian/Ubuntu, `brew install ffmpeg` on OS X, etc.).
+
+Regardless of how FFmpeg is installed, you can check if your environment path is set correctly by running the `ffmpeg` command from the terminal, as in the following example (truncated for brevity):
+
+```
+$ ffmpeg
+ffmpeg version 4.4.2 Copyright (c) 2000-2021 the FFmpeg developers
```

Comment:
Is this the -L output?

Reply:
Just `ffmpeg` also outputs this information